kms/client.go

```diff
@@ -722,6 +723,67 @@ func (c *Client) ReadDB(ctx context.Context) (*ReadDBResponse, error) {
 	}, nil
 }
 
+// Logs returns a stream of server log records from req.Host.
+// If req.Host is empty, the first host of the client's host
+// list is used.
+//
+// The LogRequest specifies which log records are fetched.
+// For example, only records with a certain log level or
+// records that contain a specific log message.
+//
+// It requires SysAdmin privileges.
+//
+// The returned error is of type *HostError.
```

Comment:
Maybe add that the returned `LogResponse` needs to be closed. Otherwise `resp.Body` will be left unclosed.